app/helpers/spree/base_helper_decorator.rb

```diff
@@ -1,4 +1,4 @@
-Spree::BaseHelper.class_eval do
+module Spree::BaseHelperDecorator
```

Comment:
@PSzyszkaSpark Module.prepend won't work with modules, please create a new helper `SpreeVolumePricing::BaseHelper` and include it in `Spree::BaseController` via decorator